lib/gui/app.js

```diff
@@ -47,6 +47,14 @@ const driveScanner = require('./modules/drive-scanner')
 const osDialog = require('./os/dialog')
 const exceptionReporter = require('./modules/exception-reporter')
 
+// Enable debug information from all modules that use `debug`
+// See https://github.com/visionmedia/debug#browser-support
+//
+// Enable drivelist debugging information
+// See https://github.com/resin-io-modules/drivelist
+electron.remote.process.env.DRIVELIST_DEBUG = /drivelist|^\*$/i.test(electron.remote.process.env.DEBUG) ? '1' : ''
+window.localStorage.debug = electron.remote.process.env.DEBUG
```

Comment:
Just a minor nit-pick, but could we move the `Enable debug information from ...` comment to just above this line?
( @jhermsmeier assuming that's the correct way that the comments line up to the code? )

Reply:
Yeah, this is fine though